src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const nav = document.querySelector('.list-item');
+const imgLarge = document.querySelector('.gallery__large-img');
+const a = document.querySelector('a');
+const img = document.querySelector('.gallery__thumb');
+
+a.addEventListener('click', () => {
+  event.preventDefault();
+});
+
+img.addEventListener('click', () => {
+  event.preventDefault();
```

Comment:
Similar to the previous comment, the `event` object is not defined within this arrow function. You need to pass `event` as a parameter to the function.